updates/control.sh

```diff
@@ -110,13 +138,14 @@ until $ntp || [[ $DHCP_STATE = 'debug' ]]; do
   sleep 1
 done
 
-#
-# rely on the DHCP server to do the right thing
-# Stick with this address until we get finished.
-#
-killall dhclient
-killall dhclient3
-
+if ! suse_ver 12; then
+  #
+  # rely on the DHCP server to do the right thing
+  # Stick with this address until we get finished.
+  #
+  killall dhclient
```

Comment:
should be fine to killall a  process that doesn't exist, right? why is the check for suse_ver 12 needed?